Makefile

```diff
@@ -2,8 +2,11 @@ PLATFORM="platform=iOS Simulator,name=iPhone 8"
 SDK="iphonesimulator"
 SHELL=/bin/bash -o pipefail
 XCODE_MAJOR_VERSION=$(shell xcodebuild -version | HEAD -n 1 | sed -E 's/Xcode ([0-9]+).*/\1/')
+IOS_EXAMPLE_PROJECT="IntegrationTests/PINCache-SPM-Integration/PINCache-SPM-Integration.xcodeproj"
+EXAMPLE_PROJECT_CARTHAGE_WORKAROUND="IntegrationTests/PINCache-SPM-Integration/PINCache-SPM-Integration.carthageWorkaround"
```

Comment:
Is EXAMPLE_PROJECT_CARTHAGE_WORKAROUND used anywhere?

Reply:
good catch! should be fixed now!